docs/articles/testing/multi-node-testing.md

```diff
@@ -96,36 +101,17 @@ We're going to show you a full code sample first and walk through how it works i
 
 First, take note of the default public constructor:
 
-```csharp
-public RestartNode2Spec() : this(new RestartNode2SpecConfig()) { }
-```
+[!code-csharp[RestartNode2Spec.cs](../../../src/core/Akka.Cluster.Tests.MultiNode/RestartNode2Spec.cs?name=PublicConstructor)]
```

Comment:
Per my previous comment, this is where we referenced source per our specifications in this document